marklogic-data-hub/src/test/java/com/marklogic/hub/dhs/DeployUserAmpsTest.java

```diff
@@ -0,0 +1,185 @@
+package com.marklogic.hub.dhs;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.client.document.DocumentManager;
+import com.marklogic.client.document.DocumentWriteSet;
+import com.marklogic.client.eval.ServerEvaluationCall;
+import com.marklogic.client.io.DocumentMetadataHandle;
+import com.marklogic.client.io.Format;
+import com.marklogic.client.io.JacksonHandle;
+import com.marklogic.client.io.StringHandle;
+import com.marklogic.hub.AbstractHubCoreTest;
+import com.marklogic.hub.HubClient;
+import com.marklogic.hub.HubConfig;
+import com.marklogic.hub.impl.Versions;
+import com.marklogic.mgmt.ManageClient;
+import com.marklogic.mgmt.ManageConfig;
+import com.marklogic.mgmt.resource.security.AmpManager;
+import com.marklogic.mgmt.resource.security.RoleManager;
+import com.marklogic.mgmt.resource.security.UserManager;
+import com.marklogic.rest.util.ResourcesFragment;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.io.IOException;
+import java.nio.file.Path;
+
+import static com.marklogic.client.io.DocumentMetadataHandle.Capability.READ;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class DeployUserAmpsTest extends AbstractHubCoreTest {
+    ObjectMapper mapper = new ObjectMapper();
+
+    @BeforeEach
+    void checkIfTestsCanBeRun(){
+        Versions.MarkLogicVersion mlVersion = versions.getMLVersion();
+        assertTrue(mlVersion.isNightly() || (mlVersion.getMajor() > 10) ||(mlVersion.getMajor() == 10 && mlVersion.getMinor() >= 440));
```

Comment:
Shouldn't this be Assumptions.assumeTrue? We don't want an error thrown if we can't test this.

Reply:
my bad, I intended it to be assumeTrue, will change